util/Setup/Program.cs

```diff
@@ -75,6 +75,10 @@ private static void Install()
             {
                 _context.Install.Domain = _context.Parameters["domain"].ToLowerInvariant();
             }
+            if (_context.Parameters.ContainsKey("dbname"))
+            {
+                _context.Install.Database = _context.Parameters["dbname"].ToLowerInvariant();
```

Comment:
I do wonder if we should not `ToLowerInvariant()` this setting in particular, while `CI` collation in SQL Server can handle any case to match a single database, because we don't necessarily control the creation of the database or the server it's deployed to, if a `CS` collation is deployed to the server's `master` database this could cause it to break if it's not used as-entered.

Reply:
I agree, removed `ToLowerInvariant()`.